DonutsGo/.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
A gitignore file is definitely necessary - it lets git know which files shouldn't be tracked and pushed into the repository (like this one) as they only have an impact on your local machine or they are too sensitive to be pushed. More details: https://git-scm.com/docs/gitignore.